src/Options/Commander.hs

```diff
@@ -30,41 +30,6 @@ Commander is an embedded domain specific language describing a command line
 interface, along with ways to run those as real programs. An complete example
 of such a command line interface is:
```

Comment:
Remove `An complete example of such a command line interface is:`, no longer applies.